OpsCenter_6.8_Release_Notes.md

```diff
@@ -1,11 +1,31 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.35
+22 Mar 2024
+
+## Backup Service
+* Fixed an issue with PIT restore when there is a schema mismatch. Added {{force_schema}} parameter to PIT Restore API to solve this issue. (OPSC-17486)
```

Comment:
`{{force_schema}}` won't render properly on GitHub markdown files. Instead, start using it with backticket (i.e. the ` character)

Reply:
ok